pcsx2/gui/AppMain.cpp

```diff
@@ -305,7 +308,7 @@ void Pcsx2App::PadKeyDispatch( const keyEvent& ev )
 	// outside of the window so the app isn't aware of the current key state.
 	m_kevt.m_shiftDown = wxGetKeyState(WXK_SHIFT);
 	m_kevt.m_controlDown = wxGetKeyState(WXK_CONTROL);
-	m_kevt.m_altDown = wxGetKeyState(WXK_MENU) || wxGetKeyState(WXK_ALT);
```

Comment:
Why remove menu?

Reply:
See commit c5f73ff53b7dce68dd28525e7d417bd540e14376 for context. Since we're checking for alt here I don't think this affects users?

Reply:
Probably isn't, I'll accept that until someone bugs me about it